stories/cosmoz-treenode-button-view.stories.ts

```diff
@@ -73,3 +73,55 @@ WithNoReset.args! = {
 	noReset: true,
 	multiSelection: false,
 };
+
+const TemplateWithPreselection = (args) => {
```

Comment:
Stories should not be used as tests. This should be converted to an actual integration test, in https://github.com/Neovici/cosmoz-treenode-navigator/blob/master/test/cosmoz-treenode-button-view.test.ts